pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
look at ``__setstate__``, BUT it looks like this was simply not handled (and therefore prob not tested), and that's why the code was commented out.

Do this. Create an old version (say 0.15) and pickle a Categorical, see if you can unpickle it with your change. (which is what the commented out code does). If that works, then checkin that pickle with a test and can call it a day.